command_line_rock_paper_scissors.py

```diff
@@ -25,6 +25,22 @@ def computer_input_to_move(game):
     return game.convert_input_to_move(random.choice(game.valid_choices))
 
 
+def players_input(human, computer):
+    players_input_store = []
+    for player in human, computer:
+        players_input_store.append('{} {}: {}'.format(player.name, 'Input', player.choice))
```

Comment:
Since the second argument to `format` is always going to be `'Input'` now, it makes more sense to put it in the string template instead.